src/main/java/Duke.java

```diff
@@ -1,10 +1,64 @@
-public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+
+/**
+ * The Duke class contains the main method to update the TaskList and
+ * execute the user's commands and input.
+ */
+class Duke {
+    public final static String LINE = "__________________________________________";
+    private static Storage storage;
+    private Ui ui;
+    private TaskList tasks;
+
+    /**
+     * The constructor for Duke takes in a String that contains the file path of the txt file
+     * to update the TaskList in the txt file.
+     *
+     * @param filePath The path which contains the txt file to be updated with the new list of tasks.
+     */
+    Duke(String filePath) {
+        ui = new Ui();
+        storage = new Storage(filePath);
+        try {
+            tasks = new TaskList(storage.load());
```

Comment:
I feel that it's odd to not have storage handle filePath and error handling with regards to loading data from the file